CHANGELOG.md

```diff
@@ -9,31 +9,30 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ## Unreleased
 
-## [25.6.4] - 2021-02-16
+## [25.6.4] - 2020-02-17
```

Comment:
What is the right version? In this changelog 25.6.4 is still present, but in the package.json it is 25.6.3? Maybe we can talk about the versioning for reverts in general.

Reply:
The patch version 25.6.4 was for SC-8613.
I removed that patch, because it is not live yet.
I added another patch to revert SC-8410. For this I created a new entry which gets version 25.6.4  again